app/main.py

```diff
@@ -149,20 +150,28 @@ def collect_item():
     Collect a tap and forward it on to XOS with the label ID.
     """
     has_tapped = HasTapped.get_or_none(has_tapped=0)
-    if has_tapped:
-        has_tapped.has_tapped = 1
-        has_tapped.save()
+    if not has_tapped:
+        return HTTPError('Tap still processing.'), abort(500, 'Tap still processing.')
```

Comment:
I don't think we should return a `500` error here, else the Lens Reader will re-queue the tap and try to send it to the Interactive Label again.

How about we either implement Tap queueing or ignore taps while we're processing previous ones?

Reply:
As discussed, implemented `tap_processing` to be explicit about this logic, and avoid unwanted requeuing of taps.

Let me know what you think.